coreMacros/src/main/scala/chisel3/SourceInfoDoc.scala

```diff
@@ -15,27 +15,23 @@ package chisel3
   *   <br>
   *
   *   The equivalent public-facing methods do not have the `do_` prefix or have the same name. Use and look at the
-  *   documentation for those. If you want left shift, use `<<`, not `do_<<`. If you want comversion to a [[Seq]] of
-  *   [[Bool]]s look at the `asBools` above, not the one below. Users can safely ignore every method in this group!
-  *   <br>
-  *   <br>
+  *   documentation for those. If you want left shift, use `<<`, not `do_<<`. If you want comversion to a
```

Comment:
should be conversion

Reply:
Fixed in the rebase. Thanks for catching this.